src/agent0/ethpy/hyperdrive/interface/read_interface.py

```diff
@@ -93,7 +93,10 @@
     from eth_account.signers.local import LocalAccount
     from eth_typing import BlockNumber, ChecksumAddress
 
-AGENT0_SIGNATURE = bytes.fromhex("a0")
+# TODO add this back in when a fix is found for extra data for morpho
```

Comment:
is it possible to use the hyperdrive name to only disable the signature for morpho?